weblate/addons/models.py

```diff
@@ -194,6 +235,54 @@ class Meta:
         prefix = ""
 
 
+def execute_addon_event(
+    addon: Addon,
+    component: Component,
+    scope: Project | Component | str,
+    event: AddonEvent,
+    method: str | Callable,
+    args: tuple | None = None,
+):
+    with transaction.atomic():
+        scope.log_debug("running %s add-on: %s", event.label, addon.name)
+        # Skip unsupported components silently
+        if hasattr(addon.addon, "can_install") and not addon.addon.can_install(
```

Comment:
Skipping should happen only for component installed add-ons:

```suggestion
        if addon.component and not addon.addon.can_install(
```